.github/workflows/checks.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     env:
       BUILD_IN_CONTAINER: false 
     container:
-      image: grafana/loki-build-image:0.32.0
```

Comment:
This is weird; howcome we're still publishing 0.31 after 0.32 was published?

Reply:
good catch, I don't know where `0.32.0` was published from as it's not mentioned in the `Makefile`, but I'm going to jump up to `0.33.0` to avoid confusion.